src/components/PageSwitcher/components/EEGEduEvoked/EEGEduEvoked.js

```diff
@@ -0,0 +1,280 @@
+import React from "react";
```

Comment:
this is just the raw module copied, with a p5 flashing ellipse sketch in the modal window, only two other lines are different below